docs/tutorials/dashboard.mdx

```diff
@@ -0,0 +1,147 @@
+import StartingJellyfishDashboard from "./common/_starting-dashboard.mdx";
+
+# The Whats and Hows of Jellyfish Dashboard
+
+## What is Jellyfish Dashboard?
+
+Jellyfish Dashboard is a web application that allows you to manage your
+Jellyfish backend. It is a React application that uses the Jellyfish React Client API
+to communicate with the backend.
+
+It was created to help Jellyfish developers in testing but was later extended and
+adapted to as a tool for tutorials and demos. Here we present a short introduction to
+the dashboard and its features.
+
+## How to start Jellyfish Dashboard?
+
+<StartingJellyfishDashboard/>
+
+## How to start Jellyfish Backend?
+
+<StartingJellyfishBackend/>
+
+## How to use Jellyfish Dashboard?
+
+Since we have Dashboard and Jellyfish backend up and running, we can dive into
+the features.
+
+### Connecting to the server
+
+We started the dashboard and the backend separately, so we need to connect them.
+
+1. Open the dashboard in your browser. Open the side menu and fill in the parameters:
+![Dashboard server connection](/img/tutorials/dashboard/dashboard_server.png)
+
+- **Server token** - a token necessary to authenticate the dashboard with the backend. For now, it's simply `development`, but
+it is meant to authenticate the dashboard to the backend, so the good practice is to use a token that is hard to guess.
+- **WS/WSS** - whether to use a secure or insecure connection. , it's `ws`, but if the backend uses secure WebSocket, use `wss`.
+- **HTTP/HTTPS** - whether to use a secure or insecure connection. By default, it's `http`, but if the backend uses secure HTTP, use `https`.
+- **Server URL** - URL of the backend server. Here, `localhost:5002` would be a local build, but it can be any URL that points to the backend.
+- **Socket Path** - path to the WebSocket endpoint. The default value is `/socket/peer/websocket`.
+
+2. Click **Connect to server**. If everything is correct, a server will appear in the list of connected servers:
+![Connected dashboard](/img/tutorials/dashboard/dashboard_connected.png)
+
+### Creating a room
+
+There are plenty of settings that You can adjust in a room:
+![Adding Room](/img/tutorials/dashboard/dashboard_add_room.png)
+
+- **h264 or vp8** - video codec used in the room. If you want to use `HLS` streaming, you need to use `h264`.
+- **Max Peers** - maximum number of peers that can join to the room.
+
+When you select the settings, click the **Add room** button. If everything is correct, a room will appear in the list of rooms:
+![Room added](/img/tutorials/dashboard/dashboard_room_added.png)
+
+### Adding and connecting peers to the room
+
+Using the **Create peer** button, you can create a peer that will be ready to connect to the room.
+![Peer created](/img/tutorials/dashboard/dashboard_peer_created.png)
+
+The peer has its `ID` and `Token`. The `ID` is used to identify the peer in the backend, and the `Token` is used to authenticate the peer with the backend.
```

Comment:
I wonder about not using `backend` term for Jellyfish. E.g.

```suggestion
The peer has its `ID` and `Token`. The `ID` is used to identify the peer in the Jellyfish Server, and the `Token` is used to authenticate the peer with the Jellyfish Server.
```

My concern is that at some point we might introduce `business logic` term and it can be missleading? Any thoughts? 

We could also introduce some shortcuts at the beggining of tutorial e.g. `Jellyfish Dashboard (JFD)` and `Jellyfish (JF)` and use those shortcuts?